extension/content.css

```diff
@@ -399,21 +399,52 @@ svg.octicon.octicon-star.dashboard-event-icon {
 }
 
 .reaction-summary-item div,
+.reaction-summary-item div a,
 .reaction-summary-item div img {
 	display: inline-block;
 }
 
+.reaction-summary-item div a {
+	transition: margin-left 0.2s;
+	padding-left: 3px;
+}
+
 .reaction-summary-item div img,
 .reaction-summary-item div span {
 	width: 20px;
 	height: 20px;
-	margin-left: 3px;
 	vertical-align: middle;
 }
+
 .reaction-summary-item div img {
+	color: #FFF; /* affects box-shadow */
+	box-shadow: 0 0 0 2px;
 	border-radius: 3px;
 }
 
+.reaction-summary-item.user-has-reacted div img {
+	color: #f2f8fa;
+}
+
+/* overlap reaction avatars when there are 5+ types of reactions */
+.reaction-summary-item:first-child:nth-last-child(n+5) a:nth-child(n+2),
+.reaction-summary-item:first-child:nth-last-child(n+5) ~ * a:nth-child(n+2) {
+	padding-left: 0;
```

Comment:
It still wouldn't match because of that needed `-2px`, but it looks good to me:

<img width="700" alt="screen shot 2017-03-07 at 00 21 48" src="https://cloud.githubusercontent.com/assets/1402241/23621164/76df2a64-024e-11e7-8c0e-88e91c2aa44d.png">

`padding-left: always 0px`